utils/fs/fs_test.go

```diff
@@ -184,6 +246,20 @@ func TestOverlayMount(t *testing.T) {
  * Private
  **********************************************************************************************************************/
 
+func createDirAndGetMountPoint(mount string, dirName string) (mountPoint string, err error) {
+	firstDir := filepath.Join(mount, dirName)
+
+	if err = os.MkdirAll(firstDir, 0o755); err != nil {
+		return "", aoserrors.Wrap(err)
+	}
+
+	if mountPoint, err = fs.GetMountPoint(firstDir); err != nil {
```

Comment:
Do not put the function under test to another function. It reduces test readability (masks the function that is tested).
```go
os.MkdirAll

fs.GetMountPoint()
```
in test looks ok for me.

Reply:
ok, got it

Reply:
Done